backend/test_observer/controllers/test_executions/models.py

```diff
@@ -124,10 +124,16 @@ def validate_review_decision(
         return review_decision
 
 
+class HistoricTestResult(BaseModel):
+    status: TestResultStatus
+    version: str
+
+
 class TestResultDTO(BaseModel):
     id: int
     name: str = Field(validation_alias=AliasPath("test_case", "name"))
     category: str = Field(validation_alias=AliasPath("test_case", "category"))
     status: TestResultStatus
     comment: str
     io_log: str
+    historic_results: list[HistoricTestResult] = []
```

Comment:
I would suggest adding a [description](https://docs.pydantic.dev/latest/concepts/json_schema/) to historic_results specifying the order (desc or asc). This would appear at the api docs

Reply:
I have now added description to this field